### DISCUSSION TITLE
727

### DISCUSSION THREAD

**Discussion by giscus** [jojoldu/blog-comments#910] [Comments]
# 727

팀에서 일을 하다보면 어떤 프로젝트가 진행될때 나도 함께 하고 싶을때가 종종 있다. 누가 봐도 커리어에, 성장에 도움이 될 것 같은 그런 프로젝트들이 진행되고 나 역시도 그런 프로젝트에 참여 하고 싶은 그런 경우 말이다. 리더가 그 프로젝트에 나를 포함시켜준다면 가장 좋겠지만, 보통은 그렇지 않고 다른 멤버가 프로젝트에 포함된다. 내가 그 프로젝트에 너무 참여하고 싶다면 어떻게 해야할까? 가장 먼저 용기를 내는 것 부터 시작이다. 리더에게 이야기를 해야한다. 그 기회를 갖고 싶다고. 하지만, 동호회가 아닌 회사라는 조직에서는 그것만으로는 기회를 얻진 못한다. 항상 일은 가장 적합한 사람에게 맡긴다. 모든 팀원이 하고 싶은 일을 하는 곳이 아니다. "잘해낼 수 있을까 걱정이 되지만 그래도 참여하고 싶다" ..

https://jojoldu.tistory.com/727

**Comment by Hoo93** [comment]
면접을 앞두고 있는 시기에 제가 부족하다는 생각을 정말 많이 했습니다. 하지만 이 글을 읽고 자신감을 얻을 수 있게 됐습니다. 부족하더라도 열심히 준비한 것은 솔직하게 다 얘기하고 오겠습니다. 항상 좋은 글 감사합니다 !

**Reply by jojoldu** [reply]
응원합니다 :)

**Comment by JoMingyu** [comment]
I can do it과 I'm able to do it은 다르지요. 전자는 의지를 표현하는 거고, 후자는 능력이 있음을 이야기하는 것이니까요. 가치를 증명하고 나야 보상이 따라오는 것이 당연하다고 생각합니다. 사람과 비교하긴 좀 그렇지만, 물건도 가치가 있어야 구매하고 싶으니까요. '적게 주니 적게 일한다'는 마인드에 동의하고 살았던 적이 있는데, 그게 아주 잠깐이었던 게 참 다행입니다.

이번에도 좋은 글 감사합니다!

**Comment by SEOKKAMONI** [comment]
"용기만 내면 준비가 안된 사람에게도 엄청난 능력이 생기거나 하는 일은 현실에 없다."라는 말에 한방 얻어맞은 것 같네요 준비가 된 사람이 되기 위해 더 노력해야겠네요 좋은 글 감사합니다.

**Comment by Chadolbaegi128** [comment]
오늘도 좋은 글 잘 보고 갑니다. 결과가 없는 원대한 이상보다 사소하지만 결실이 있는 능력이 훨씬 중요하다는 것을 깨달았네요

**Comment by chrishoonS** [comment]
좋은글 잘 읽어보고 갑니다. 아직 저의 능력 부족이겠지만, 현실은 생각보다 용기를 내어도 기회가 주어지지 않는 경우도 있고, 데드라인을 지키기 위해 외주나 높으신 분들이 프로젝트를 구성하는 경우가 많은것 같더라구요. 반복적이고 단순한 일들을 많이 맡아 하다보니, 프로젝트를 새롭게 구성하는 방법이라던지, 미니 버전 구현을 어떻게 해야할지도 감이 잘 안잡히네요. 그냥 가만히 기다리기 보다는, 글쓰신 것처럼 혼자서라도 이것저것 찾아보며 저만의 프로토타입을 만들어보는 시간들이 필요하겠죠?ㅠㅠ

**Comment by sa02045** [comment]
좋은 글 감사합니다. 언제나 늘 잘 보고있습니다

**Comment by InhwanJeong** [comment]
회사에는 열심히 하는 사람은 정말 많습니다. 하지만 잘하는 사람과 도전하는사람은 많지 않습니다.
저는 잘하는 사람이 되고자 노력했는데, 도전하는 사람이 될 수 있는 용기도 배워갑니다 ^^ 감사합니다.

**Comment by jobum97** [comment]
감명 깊은 글이였습니다 감사합니다
